pkg/controller.v1beta1/consts/const.go

```diff
@@ -50,11 +50,11 @@ const (
 	ConfigTrialResources = "trial-resources"
 
 	// LabelExperimentName is the label of experiment name.
-	LabelExperimentName = "experiment"
+	LabelExperimentName = "katib.kubeflow.org/experiment"
 	// LabelSuggestionName is the label of suggestion name.
-	LabelSuggestionName = "suggestion"
+	LabelSuggestionName = "katib.kubeflow.org/suggestion"
 	// LabelDeploymentName is the label of deployment name.
-	LabelDeploymentName = "deployment"
+	LabelDeploymentName = "katib.kubeflow.org/deployment"
```

Comment:
Will it be released in next version?

Reply:
I think it's fine to release it in 0.12, since it's not breaking change.
These labels are used internally in Katib Controller.

Reply:
SGTM.

/lgtm